hr_expense_operating_unit/models/hr_expense.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016-19 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).).
+
+from odoo.tools.translate import _
+from odoo import api, fields, models
```

Comment:
Improve `from odoo import api, fields, models, _` and remove https://github.com/OCA/operating-unit/pull/184/files#diff-85ed473ab0a9e52c02ccb1eb7ea9aa0bR5